ecms_base/modules/custom/ecms_workflow/ecms_workflow.module

```diff
@@ -17,18 +17,18 @@ use Drupal\webform\WebformInterface;
  * this will ensure they are assigned to the proper workflow.
  */
 function ecms_workflow_entity_bundle_create($entity_type_id, $bundle) : void {
-  if ($entity_type_id !== 'node') {
-    return;
+  // If we are dealing with nodes, add the correct permissions.
+  if ($entity_type_id === 'node') {
+    // Call the workflow service to update configuration.
+    \Drupal::service('ecms_workflow.bundle_create')
+      ->addContentTypeToWorkflow($bundle);
   }
 
-  // Notifications are managed using the ecms_api module.
-  if ($bundle === 'notification') {
```

Comment:
Since we are no longer guarding against notifications here, we should do so in the addContentTypeToWorkflow method

Reply:
Done.